### PR TITLE
Add db persistence for cardsets

### PR DESCRIPTION
**Pull request by allenwhitedev**

### CODE REVIEW

File: src/components/CustomSetBuilder/BulkAdd/BulkAddForm.tsx [hunk 1]
```diff
@@ -36,15 +37,15 @@ function BulkAddForm(props: ParentProps) {
         if(isQuickCreate && !set){
             const cardNames = cardList.split(/\r?\n/);
             const setId = getSetId(setName)
-            dispatch(addSet({id: setId, set_name: setName, set_code: setName, num_of_cards: cardNames.length, tcg_date: Date(), custom_set: true}))
+            dispatch(addSet({id: setId, set_name: setName, set_code: setName, num_of_cards: cardNames.length, tcg_date: Date(), custom_set: true, author: ip}))
```
Comment (danielschneider22) [inline src/components/CustomSetBuilder/BulkAdd/BulkAddForm.tsx:40]: author should come from state email address

Comment (danielschneider22) [reply src/components/CustomSetBuilder/BulkAdd/BulkAddForm.tsx:40]: getUserEmail

Comment (danielschneider22) [reply src/components/CustomSetBuilder/BulkAdd/BulkAddForm.tsx:40]: jk get from cookie in backend. however, we should not be able to do cardset stuff without being logged in

**Comment by allenwhitedev** [comment]
Closing, changes were manually recreated in https://github.com/danielschneider22/Yugiohdrafter/pull/10

**Pull request closed without merging**